website/docs/commands/compile.md

```diff
@@ -17,8 +17,8 @@ scala-cli compile Hello.scala
 ```
 
 Note that most Scala CLI commands automatically compile your code, if necessary.
-The `compile` command is useful if you'd like to check that your code compiles,
-or know of compilation warnings, without running it or packaging it.
+The `compile` command is useful if you want to check that your code compiles
+(or to see the compilation warnings) without running it or packaging it.
```

Comment:
NIT
```suggestion
(or to see the compilation warnings, if any) without running it or packaging it.
```

Reply:
changed to `(or to see the compilation warnings, if any occur) without running it or packaging it.`